docs/usage/exposureclasses.md

```diff
@@ -0,0 +1,92 @@
+See [here](#Gardenlet-Configuration-ExposureClass-handlers).
+
+
+The `RuntimeClass` support via its `.scheduling` section also the selection of a node subset (which have the respective controller runtime installed) for pod scheduling.
+The `ExposureClass` support also via the `.scheduling` section the selection of a subset of available Seed clusters whose Gardenlet is capable to apply the exposure configuration for the Shoot control plane accordingly.
+
+### Usage by a Shoot
+
+A `Shoot` can reference an `ExposureClass` via the `.spec.exposureClassName` field.
+
+**Attention:** When creating a Shoot cluster the Gardener scheduler will try to assign the Shoot cluster to a Seed cluster which will host its control plane.
+The scheduling behaviour can be influencd by the Shoots `seedSelectors` and/or `tolerations`.
+ExposureClasses can contain also scheduling instructions and if a Shoot is referencing an ExposureClass then the scheduling instructions of both will be merged into the Shoot.
+Those union of scheduling instructions might lead to a selection of a Seed which is not able to deal with the `handler` of the ExposureClass and the Shoot creation might end up in a creation error.
+
+### Gardenlet Configuration ExposureClass handlers
+
+The Gardenlet is responsible to realize the control plane exposure strategy defined in the referenced `ExposureClass` of a `Shoot`.
+
+Therefore the `GardenletConfiguration` can contain an `.exposureClassHandlers` list with the respective configuration.
+
+Example of the `GardenletConfiguration`:
+```yaml
+exposureClassHandlers:
+- name: internetconfig
+  loadBalancerService:
+    annotations:
+      loadbalancer/nework: internet
+- name: internalconfig
+  loadBalancerService:
+    annotations:
+      loadbalancer/nework: internal
+  sni:
+    ingress:
+      namespace: ingress-internal
+      labels:
+        network: internal
+```
+
+Each Gardenlet can define how the handler of a certain ExposureClass need to be implemented for the Seed(s) where its reponsible for.
+
+The `.name` is the name of the handler config and it need to match to the `.handler` in the `ExposureClass`.
+
+All control planes on a Seed are exposed via a load balancer.
+Either a dedicated one or a central shared one.
+The load balancer service need to be configured in a way that it is reachable from the target network enviroment.
+Therefore the configuration of load balancer service need to be conifgurable and this can be done via the `.loadBalancerService` section.
+The common way to influence load balancer service behaviour is via annotations where the respective cloud-controller-manager will react on and configure the infrastucture load balancer accordingly.
+
+In case the Gardenlet runs with activated `APIServerSNI` feature flag the control planes on a Seed will be exposed via a central load balancer and with Envoy with TLS SNI pass through.
```

Comment:
```suggestion
In case the Gardenlet runs with activated `APIServerSNI` feature flag (default), the control planes on a `Seed` will be exposed via a central load balancer and with Envoy with TLS SNI pass through.
```